Rawl_B502/de_peritia_&_de_genealogis_dál_niad_cuirp_incipit.ttl

```diff
@@ -0,0 +1,755 @@
+    rel:childOf <#Tailech>;
+    rdfs:comment "moccu Tellaich";
+    rdfs:seeAlso <https://www.wikidata.org/wiki/Q622095>;
+    owl:sameAs <http://example.com/LL/senchas_coitechenn_clainde_fergusa.ttl#Finnian>.
+
+<#Rígnach>
+    a foaf:Person;
+    irishRel:nomName "Rígnach";
+    rdfs:comment "nóebh-uaga Úa Lóscán";
+    rdfs:comment "h-i Cill Rígnaige h-i Fothartaib Mara".
+
+<#CathairMár-82bed530>
+    a foaf:Person;
+    irishRel:nomName "Cathair Már";
+    irishRel:numChild 30;
+    owl:sameAs <#CathairMár>;
+    rdfs:comment "Do-róebdatar tantum sed .x.".
+
+<#FiachuBah-Aiccid-7b5eae70>
+    a foaf:Person;
+    irishRel:nomName "Fiachu Ba h-Aiccid";
+    rdfs:comment "a quo sunt reges, Úi Ceinselaig & Úi Dúnlainge";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <#FiachuBah-Aiccid>;
+    rel:childOf <#CathairMár-82bed530>.
+
+<#DáreBarrach>
+    a foaf:Person;
+    irishRel:nomName "Dáre Barrach";
+    rdfs:comment "a quo Úi Bairrche";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#DareBarrach>.
+
+<#CrimthannÁn>
+    a foaf:Person;
+    irishRel:nomName "Crimthann Án";
+    rdfs:comment "a quo Úi Chrimthainn";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#DareBarrach>.
+
+<#RusFailgech>
+    a foaf:Person;
+    irishRel:nomName "Rus Failgech";
+    rdfs:comment "a quo Úi Failgi";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#RosFalge>.
+
+<#BresalEnechglas>
+    a foaf:Person;
+    irishRel:nomName "Bresal Enechglas";
+    rdfs:comment "a quo Úi Enechglais";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#BressalEnechglas>.
+
+<#EochuTimmíne>
+    a foaf:Person;
+    irishRel:nomName "Eochu Timmíne";
+    rdfs:comment "a quo Úi Timmíne";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#EochoTimmin>.
+
+<#AilillCéthech>
+    a foaf:Person;
+    irishRel:nomName "Ailill Céthech";
+    rdfs:comment "a quo Úi Chéthig";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#AilillCethech>.
+
+<#FergusLuascán-ee994a40>
+    a foaf:Person;
+    irishRel:nomName "Fergus Luascán";
+    rdfs:comment "ut supra diximus";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <#FergusLuascán>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FergusLuascan>.
+
+<#Dergmosach>
+    a foaf:Person;
+    irishRel:nomName "Dergmosach";
+    rdfs:comment "a quo Úi Dergmosaich h-ic Áth Chliath";
+    rdfs:comment "Ór a n-díre enechclainne";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Dercmossach>.
+
+<#Fiachu>
+    a foaf:Person;
+    irishRel:nomName "Fiachu";
+    owl:sameAs <#FiachuBah-Aiccid-7b5eae70>;
+    rdfs:comment "De quibus dicitur: [verse]".
```

Comment:
Yes. Hence the square brackets? Unfortunately, many of the quoted quatrains that appear in the genealogies have no URL or anything to allow us to link to them via "rdfs:seeAlso". So we need to indicate the provenance of the material they contain somehow.

Is there some way we could easily create a URL or URI for this sort of material? It doesn't have to be a big elaborate database of cited verse, just a quick reference list.